x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -294,6 +298,34 @@ export type AnomalyAlertSeverityType = ValuesType<
   typeof ANOMALY_ALERT_SEVERITY_TYPES
 >['type'];
 
+export const ANOMALY_ALERT_DETECTORS = [
+  {
+    type: 'latency',
```

Comment:
Please use the existing constants

```suggestion
    type: ApmMlDetectorType.txLatency,
```